benchmarks/DeepLearning/Models/Resnet18/CMakeLists.txt

```diff
@@ -0,0 +1,168 @@
+add_custom_command(
+  OUTPUT 
+    ${CMAKE_CURRENT_SOURCE_DIR}/forward.mlir 
+    ${CMAKE_CURRENT_SOURCE_DIR}/subgraph0.mlir
+  COMMAND
+    python3 ${CMAKE_CURRENT_SOURCE_DIR}/buddy_resnet18_import.py
+  COMMENT "Generating forward.mlir, subgraph0.mlir..."
+)
+
+add_custom_command(
+  OUTPUT forward_auto_vectorization.o
+  COMMAND
+    cat ${CMAKE_CURRENT_SOURCE_DIR}/forward.mlir |
+    sed -e {s/@forward/@forward_auto_vectorization/}
+        -e {s/@subgraph0/@subgraph0_auto_vectorization/} |
+    ${LLVM_MLIR_BINARY_DIR}/mlir-opt 
+    -pass-pipeline
+    "builtin.module(func.func(tosa-to-linalg-named, tosa-to-linalg, tosa-to-tensor, tosa-to-arith), \
+              empty-tensor-to-alloc-tensor, convert-elementwise-to-linalg, arith-bufferize, \
+              func.func(linalg-bufferize, tensor-bufferize), func-bufferize)" |
+    ${LLVM_MLIR_BINARY_DIR}/mlir-opt
+    -pass-pipeline
+    "builtin.module(func.func(buffer-deallocation-simplification, convert-linalg-to-loops), \
+              eliminate-empty-tensors, func.func(llvm-request-c-wrappers), \
+              convert-math-to-llvm, convert-math-to-libm, convert-scf-to-cf, \
+              convert-arith-to-llvm, expand-strided-metadata, finalize-memref-to-llvm, \
+              convert-func-to-llvm, reconcile-unrealized-casts)" |
+    ${LLVM_MLIR_BINARY_DIR}/mlir-translate -mlir-to-llvmir |
+    ${LLVM_MLIR_BINARY_DIR}/llc -O3
+      -mtriple=${BUDDY_OPT_TRIPLE} -mattr=${BUDDY_OPT_ATTR} -filetype=obj
+      -o ${CMAKE_CURRENT_BINARY_DIR}/forward_auto_vectorization.o
+  DEPENDS
+    ${CMAKE_CURRENT_SOURCE_DIR}/forward.mlir
+  COMMENT "Building forward_auto_vectorization.o"
+  VERBATIM)
+
+add_custom_command(
+  OUTPUT subgraph0_auto_vectorization.o
+  COMMAND
+    cat ${CMAKE_CURRENT_SOURCE_DIR}/subgraph0.mlir |
+    sed -e {s/@subgraph0/@subgraph0_auto_vectorization/} |
+    ${BUDDY_MLIR_BINARY_DIR}/buddy-opt 
+      -pass-pipeline 
+      "builtin.module(func.func(tosa-to-linalg-named),func.func(tosa-to-linalg),func.func(tosa-to-tensor),func.func(tosa-to-arith))" |
+    ${BUDDY_MLIR_BINARY_DIR}/buddy-opt
+    -convert-elementwise-to-linalg
+    -func-bufferize-dynamic-offset
+    -arith-bufferize
+    -func-bufferize
+    -tensor-bufferize
+    -linalg-bufferize
+    -finalizing-bufferize
+    -convert-linalg-to-loops
+    -lower-affine
+    -convert-scf-to-cf
+    -llvm-request-c-wrappers
+    -convert-math-to-llvm
+    -convert-math-to-libm
+    -convert-arith-to-llvm
+    -convert-func-to-llvm
+    -expand-strided-metadata
+    -finalize-memref-to-llvm
+    -reconcile-unrealized-casts |
+    ${LLVM_MLIR_BINARY_DIR}/mlir-translate -mlir-to-llvmir |
+    ${LLVM_MLIR_BINARY_DIR}/llc -O3
+      -mtriple=${BUDDY_OPT_TRIPLE} -mattr=${BUDDY_OPT_ATTR} -filetype=obj
+      -o ${CMAKE_CURRENT_BINARY_DIR}/subgraph0_auto_vectorization.o
```

Comment:
Keep the code standards (e.g. indentation) of this part consistent with the previous code.